var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -33,9 +33,10 @@ class Openmpi(AutotoolsPackage):
     version('master', branch='master')
 
     # Current
-    version('4.0.4', sha256='47e24eb2223fe5d24438658958a313b6b7a55bb281563542e1afc9dec4a31ac4')  # libmpi.so.40.20.4
+    version('4.0.5', sha256='c58f3863b61d944231077f344fe6b4b8fbb83f3d1bc93ab74640bf3e5acac009')  # libmpi.so.40.20.5
 
     # Still supported
```

Comment:
Based on the homepage, 4.0 is current and 3.1/3.0 are still supported. Should all 4.0.X releases go under current?

Reply:
@adamjstewart check now